packages/core/src/App/Containers/Modals/trading-assessment-existing-user.jsx

```diff
@@ -0,0 +1,174 @@
+                title={localize('Risk Tolerance Warning')}
+                button_text={localize('Yes, I understand the risk.')}
+                onClick={handleAcceptRisk}
+                body_content={
+                    <Localize
+                        i18n_default_text='CFDs and other financial instruments come with a high risk of losing money rapidly due to leverage. You should consider whether you understand how CFDs and other financial instruments work and whether you can afford to take the high risk of losing your money. <0/><0/> To continue, you must confirm that you understand your capital is at risk'
+                        components={[<br key={0} />]}
+                    />
+                }
+            />
+        );
+    } else if (should_show_appropriateness_test_warning_modal) {
+        return (
+            <TestWarningModal
+                show_risk_modal={should_show_appropriateness_test_warning_modal}
+                body_content={
+                    <Text as='p' size='xs'>
+                        <Localize
+                            i18n_default_text='In providing our services to you, we are required to ask you for some information to assess if a given product or service is appropriate for you and whether you have the experience and knowledge to understand the risks involved.<0/><1/>'
+                            components={[<br key={0} />, <br key={1} />]}
+                        />
+                        <Localize
+                            i18n_default_text='Based on your answers, it looks like you have insufficient knowledge and experience in trading CFDs. CFD trading is risky and you could potentially lose all of your capital.<0/><0/>'
+                            components={[<br key={0} />, <br key={1} />]}
+                        />
+                        <Localize i18n_default_text='Please note that by clicking ‘OK’, you may be exposing yourself to risks. You may not have the knowledge or experience to properly assess or mitigate these risks, which may be significant, including the risk of losing the entire sum you have invested' />
+                    </Text>
+                }
+                footer_content={
+                    <React.Fragment>
+                        <Button
+                            type='button'
+                            large
+                            text={localize('OK')}
+                            primary
+                            onClick={handleAcceptAppropriatenessTestWarning}
+                        />
+                    </React.Fragment>
+                }
+            />
+        );
+    } else if (should_show_trade_assessment_form) {
+        return (
+            <React.Fragment>
+                <DesktopWrapper>
+                    <Modal
+                        is_open={should_show_trade_assessment_form}
+                        title={localize('Trading Experience Assessment')}
+                        width='904px'
```

Comment:
This has to be exactly 904 by 688 pixels?

Reply:
Thats the size of modal defined in the figma `https://www.figma.com/file/yoAD2CiEmlt2x2sPmKWpoG/MF-onboarding-(Classic-UI)?node-id=7665%3A135261`